### ISSUE TITLE
overlapping + sign in add users

### ISSUE THREAD

**Issue by eoglethorpe** [the-deep/client#606]
![image](https://user-images.githubusercontent.com/9661732/51221204-885ff900-1906-11e9-9fe6-ccf9aababe4a.png)

**Comment by tnagorra** [comment]
That is the intended behavior and we have already talked about this before.
If you don't like the absolute positioning, the other alternative would be:
1. Always leave a space for button (will have weird empty space when button is not shown)
2. Button occupies space after it is shown (The height of the element may increase as email/name won't fit in single line)

**Comment by eoglethorpe** [comment]
Yes we've discussed and maybe there was miscommunication, but i don't recall being ok with the overlap. I think we can leave it overlapping if it we make the button only appear on hover or having it very faded if it's not being hovered over?

I agree that it's a lot of space to give up for the button
